### DISCUSSION TITLE
Repos and Topics abused to advertise game hacks/cheats

### DISCUSSION THREAD

**Discussion by EtaoinWu** [community/community#79042] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

**Before clicking on any links from the repos mentioned in this post, please check their URLs, as the linked websites may be harmful for your browser/device.**

Under topic [#cs2-hack-download](https://github.com/topics/cs2-hack-download), we can see a number of repos containing README.md advertising some Counter-Strike 2 hacks. These repos are created by obviously fake accounts with generic `<word><word><numbers>` usernames. Some repos [(1)](https://github.com/ReplyHunger/CS-2-cheat-download) are basically empty with just a README, while some others [(2)](https://github.com/nathanfa/PowerNukkit-Pterodactyl-Egg) are forks of existing repos with README replaced.

This essentially turned some topic pages (like [1](https://github.com/topics/cs2-skin-changer-download), [2](https://github.com/topics/cs2-aimbot), [3](https://github.com/topics/cs-2-hack)) into advertisements of (potentially dangerous) cheats. Do note that, however, there are legitimate development repos of cheats (that may not violate GitHub's TOS, but may violate Valve's) on GitHub, like the most starred ones [here](https://github.com/topics/cs2-cheat).

I opt to not report the users, but make a complain here, because there may be too many such bot accounts, and creating some new ones is just as easy for the abusers. Hopefully GitHub can find a way to mitigate this. If this is not stopped effectively, I am afraid that the abusers may start to use these bot accounts to star each other, boosting their places and outrank legitimate repos.

**Comment by pkr-sadx** [comment]
I bet all these accounts use 2FA so it should be safe huh!


F GitHub

**Comment by queenofcorgis** [comment, marked as answer]
Hi @EtaoinWu  👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools rather than posting in the community where folks may accidentally click on the links. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!

**Comment by Kuravar** [comment]
I'm trying to figure out how to play cs go. I understand that the game is simple, but I just can’t win. Are there any hacks or tips?

**Reply by Fishrot** [reply]
There are a lot of different tips, articles and resources for cs go players, where you can find a lot of information about strategies and tactics. I think you should just google it. Honestly, now I'm more passionate about cs go skins. [Here it is](https://lyncconf.com/awp-atheris-is-an-affordable-sniper-rifle-in-cs-go/) interesting to read more about cs go skin trading, and it’s cool that now you can buy and sell cs go skins safely for real money.

**Comment by Leslie-lang** [comment]
Hi, thanks for surfacing this issue. You're absolutely right, bots are abusing GitHub topics and README files to advertise potentially harmful Counter-Strike 2 hacks, and the generic fake account names make them easy to spot but hard to stop. Since they can just create new accounts, reporting individually isn't sustainable. GitHub really needs smarter automated detection, like flagging repos that consist of nothing but a README with shady external links, or requiring some minimum activity before allowing topic tags. It's especially concerning because these could push legitimate cheat development repos (which are already a gray area) out of search results. On a completely different note, if anyone's looking for a safe and fun gaming experience without security risks, you might enjoy the [3s game](https://apkrain.com.pk/3s-game/). Hope GitHub takes meaningful action soon. Thanks!

**Comment by ik6360165-droid** [comment]
I agree this is becoming a serious issue. Topic pages should help developers discover useful projects, not become flooded with fake repositories promoting potentially harmful downloads or cheats.

It would be helpful if GitHub improved automated detection for newly created bot accounts, duplicate README patterns, and suspicious topic abuse. Community reporting is useful, but stronger preventive measures would make these topic pages much more trustworthy.

As someone who manages a legitimate Android app download website, I think users should always verify the source before downloading any APK. I follow the same approach on my own site,[ Six Games APK](https://sixgames.net.pk/), by focusing on clear installation instructions and safe download practices rather than misleading promotions. Hopefully GitHub continues improving its moderation against this type of abuse.
